### PR TITLE
Move and expand description of design doc process into README

### PR DESCRIPTION
**Pull request by anorth**
There are some other notes about similar processes elsewhere (some internal) that I will subsequently remove. I think that next to the design docs themselves is the best place for this.

### CODE REVIEW

File: designdocs.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 ## Goal
 
-Articulate the reasons why we write designdocs and propose that the definition of a plan is that it is written down.
+Articulate what a designdoc is an reasons why we write and publish them.
```
Comment (acruikshank) [inline designdocs.md]: an -> and

File: README.md [hunk 2]
```diff
@@ -17,7 +19,35 @@ _Explainer talks or demos._
...
+  * Use `DRAFT`, `IN REVIEW`, `IMPLEMENTING` near the top of the content to note status.
+* When you're ready, enable public comments. As of April 2019, this takes 5 clicks:
+  * Click "Share" button.
+  * Click "Who has access" to open the advanced settings panel.
+  * Toggle on "Link sharing".
+  * Choose "People at Filecoin with the link can *comment*" from dropdown.
+  * Check the checkbox for "Allow external access".
+* Add an entry with shareable link in this README's list of design docs, thus making the document public (pull request not necessary).
+* Email a link to the `dev@filecoin.org` email list, which will reach all committers. Also announce on the filecoin-project Slack.
+  * This email thread will function as as a high-signal channel for high-level discussion of the proposal.
```
Comment (mishmosh) [inline README.md]: Lines 36-41 are quite long, some suggestions inline to clarify or simplify

File: README.md [hunk 3]
```diff
@@ -17,7 +19,35 @@ _Explainer talks or demos._
...
+* Solicit feedback from individuals and/or channels, evolving the design as needed.
+  * Advertise major changes with a post to the email/slack threads where the design was announced.
+  * Seek maintainer blessing before considering the proposal accepted.
+* Once ready, translate into GitHub/ZenHub epics or issues according to the appropriate repo's contributing guide.
+  * Close the issue that called for the design doc to be written in the first step above.
```
Comment (mishmosh) [inline README.md]: I think this can be removed. Closing an issue is pretty routine.

File: README.md [hunk 4]
```diff
@@ -17,7 +19,35 @@ _Explainer talks or demos._
...
+  * Click "Who has access" to open the advanced settings panel.
+  * Toggle on "Link sharing".
+  * Choose "People at Filecoin with the link can *comment*" from dropdown.
+  * Check the checkbox for "Allow external access".
+* Add an entry with shareable link in this README's list of design docs, thus making the document public (pull request not necessary).
+* Email a link to the `dev@filecoin.org` email list, which will reach all committers. Also announce on the filecoin-project Slack.
+  * This email thread will function as as a high-signal channel for high-level discussion of the proposal.
+* Solicit feedback from individuals and/or channels, evolving the design as needed.
+  * Advertise major changes with a post to the email/slack threads where the design was announced.
+  * Seek maintainer blessing before considering the proposal accepted.
```
Comment (mishmosh) [inline README.md]: Could combine 37-39 into: Solicit feedback from individuals and/or channels, evolving the design as needed. Solicit approval from [number] maintainer[s] before moving to `Accepted`.

**Pull request closed without merging**